client/blocks/checklist/checklist-task.jsx

```diff
@@ -56,8 +56,8 @@ export class ChecklistTask extends PureComponent {
 			duration,
 			title,
 			translate,
+			buttonText,
```

Comment:
The existing implementation has a fallback value, it would be a good idea to keep that so we don't unexpectedly break anything 🙂 

We can just restore the previous default right here:

```js
buttonText = translate( 'Do it!' ),
```